Makefile

```diff
@@ -1,3 +1,18 @@
+# The following parameters are required:
+# - AWS_REGION
+# - AWS_ACCESS_KEY
+# - AWS_SECRET_ACCESS_KEY
+# - GITHUB_ORG
+# - GITHUB_REPO
+# - CIRCLECI_TOKEN
+#
 
 run:
-	@echo Done
+	@echo "Set CIRCLECI environment variables\n"
+	curl -X POST --header "Content-Type: application/json" -d '{"name":"AWS_REGION", "value":"${AWS_REGION}"}' https://circleci.com/api/v1.1/project/github/${GITHUB_ORG}/${GITHUB_REPO}/envvar?circle-token=${CIRCLECI_TOKEN}
```

Comment:
These aren't quite the right env vars for this project. In the deploying with Zero doc it lists the env vars as:
```
AWS_ACCESS_KEY_ID
AWS_SECRET_ACCESS_KEY
SLACK_WEBHOOK
```

Reply:
We used the params needed by the `sync` command in the `aws-s3` orb: https://circleci.com/orbs/registry/orb/circleci/aws-s3#commands-sync

We've also removed the dependency on SLACK_WEBHOOK for now because it wasn't being used anywhere.